lib/showoff.rb

```diff
@@ -1095,13 +1163,14 @@ def inline_js(jses, pre = nil)
     end
 
     def inline_all_js(jses_directory)
-       inline_js(Dir.entries(File.join(File.dirname(__FILE__), '..', jses_directory)).find_all{|filename| filename.length > 2 }, jses_directory)
```

Comment:
There appear to be a lot of extraneous whitespace changes. Did you accidentally reindent the whole file?